src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -117,6 +129,8 @@ export class CalcitePickList {
       selectedValues.delete(value);
     }
     this.lastSelectedItem = item;
+    this.calciteListSelectionChange.emit(selectedValues);
+    // TODO: remove in the next major version bump.
```

Comment:
Should we create an issue for stuff going away in a major release? It'd help keep everything in one place and easier to track.

Reply:
Can we just add a `@deprecated` comment? then we can just search for things that have `@deprecated`

Reply:
sure. added @deprecated.